source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -94,13 +94,7 @@ It is used at the data acquisition and data preparation stages to:
   device's local memory. Therefore, a data source enables complex data analytics
   scenarios, such as :capterm:`online computations <Online mode>`.
 
-- Filter and normalize :capterm:`feature` values that are being extracted.
-
-- Recover missing :capterm:`feature` values.
-
-- Detect :capterm:`outliers <outlier>` and recover the abnormal data.
-
-- Transform datasets into numerical representation. Data source shall
+- To transform datasets into numerical representation. Data source shall
```

Comment:
other items in the list don't start with "to"
```suggestion
- Transform datasets into their numerical representation. Data source shall
```

Reply:
Done.